docs/ww-manual/resident-experience.md

```diff
@@ -0,0 +1,95 @@
+
+### Downloading your bin schedule
+
+Bin day collection schedules can be downloaded from the site by selecting ‘Download your collection schedule’ from the menu on the right-hand side. Residents can also add schedules to their calendar.
+
+<img alt="Downloading your bin schedule" src="/assets/img/ww-user-guide/WasteWorks-download-bin-collection-schedule-resident-experience.png" class="admin-screenshot" />
+
+### Requesting a new bin
+
+From the main ‘Your bin days’ page, residents can select ‘Request a new container’ from the ‘More services’ menu.
+
+<img alt="Requesting a new bin" src="/assets/img/ww-user-guide/WasteWorks-request-new-container-resident-experience.png" class="admin-screenshot" />
+
+They can then select the type of container they want to request and hope many they need.
+
+<img alt="Select container" src="/assets/img/ww-user-guide/WasteWorks-select-container-resident-experience.png" class="admin-screenshot" />
+
+
+They will then be asked for some basic information before being able to submit the request.
+
+### Reporting a missed collection
+
+To report a missed collection, from the main ‘Your bin days’ page the resident will be asked to select the bin in question and provide their contact details so that updates can be provided.
+<img alt="Report a missed collection" src="/assets/img/ww-user-guide/WasteWorks-report-missed-collection.png" class="admin-screenshot" />
+
+They will then be able to review the information they have provided before submitting the report.
+
+### Subscribing to a bulky or green waste collection
+
+From the main ‘Your bin days’ page, residents can select a subscription option (eg bulky or green garden waste) from the ‘More services’ menu.
+
+<img alt="Subscribe to collections" src="/assets/img/ww-user-guide/WasteWorks-subscribe-to-waste-collections.png" class="admin-screenshot" />
+
+After a summary of subscription information, the resident will be able to say whether they already have any bins and how many collections to which they would like to subscribe.
+
+<img alt="Subscription bin info" src="/assets/img/ww-user-guide/WasteWorks-subscription-bin-info.png" class="admin-screenshot" />
+
+If you have enabled more than one payment method, the resident will be able to select their preference and input their contact details.
+
+<img alt="Choose your subscription payment option" src="/assets/img/ww-user-guide/WasteWorks-subscription-payment.png" class="admin-screenshot" />
+
+The resident will then be shown a summary of the subscription details before continuing to payment, which will be facilitated by whichever payment provider you use.
+
+<img alt="Subscription summary" src="/assets/img/ww-user-guide/WasteWorks-subscription-summary.png" class="admin-screenshot" />
+
+The payment will be handled within the WasteWorks workflow, the resident will not need to be redirected to another site or service.
```

Comment:
You are redirected to the payment provider, so this might need rewording.

Reply:
Also corrected this